tests/neo-vm.Tests/Types/InteropService.cs

```diff
@@ -9,21 +9,6 @@ public bool Invoke(byte[] method, ExecutionEngine engine)
         {
             switch (Encoding.ASCII.GetString(method))
             {
-                case "System.ExecutionEngine.GetEntryScriptHash":
```

Comment:
This syscall dissapear?

Reply:
https://github.com/neo-project/neo/blob/976659a5778a03f5aef88dcce56122eb471557f4/neo/SmartContract/StandardService.cs#L38-L40

They are in neo.